src/EditorFeatures/Core/Implementation/CallHierarchy/CallHierarchyDetail.cs

```diff
@@ -102,7 +102,8 @@ public void NavigateTo()
             if (document != null)
             {
                 var navigator = _workspace.Services.GetService<IDocumentNavigationService>();
-                navigator.TryNavigateToSpan(_workspace, document.Id, _span, usePreviewTab: true);
+                var options = _workspace.Options.WithChangedOption(NavigationOptions.UsePreviewTab, true);
+                navigator.TryNavigateToSpan(_workspace, document.Id, _span, options);
```

Comment:
its hard to understand what the argument `true` represents here. perhaps named args?